src/table/components/TableWrapper.jsx

```diff
@@ -85,16 +91,23 @@ export default function TableWrapper(props) {
   useEffect(() => {
     const memoedContainer = tableContainerRef.current;
     const memoedWrapper = tableWrapperRef.current;
-    if (!memoedContainer || !memoedWrapper) return () => {};
-
+    const tablePaginationSection = tablePaginationSectionRef.current;
+    if (!memoedContainer || !memoedWrapper || !tablePaginationSection) return () => {};
+    const handleResize = () => {
+      setPaginationHeight(tablePaginationSection.clientHeight);
+    };
+    handleResize();
```

Comment:
should this run here?

Reply:
Yes, it needs to be run at the first time